script/app-mlperf-inference-mlcommons-python/meta.yaml

```diff
@@ -546,9 +546,11 @@ deps:
     enable_if_env:
       MLC_MODEL:
         - pointpainting
-    skip_if_env:
+    skip_if_any_env:
       MLC_RUN_STATE_DOCKER:
         - "yes"
+      MLC_USE_DATASET_FROM_HOST:
```

Comment:
Inside a docker run, if the checkpoint path is set, we skip the download in the model download script right?